pkg/controller/backup_sync_controller_test.go

```diff
@@ -76,6 +78,28 @@ func TestBackupSyncControllerRun(t *testing.T) {
 			},
 			namespace: "heptio-ark",
 		},
+		{
+			name: "normal case dup",
+			cloudBackups: []*v1.Backup{
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-1").Backup,
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-1").Backup,
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-3").Backup,
+			},
+			namespace: "ns-1",
+		},
+		{
+			name: "normal case with an existing client backups",
+			cloudBackups: []*v1.Backup{
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-1").Backup,
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-2").Backup,
+				arktest.NewTestBackup().WithNamespace("ns-1").WithName("backup-3").Backup,
+			},
+			existingBakcups: map[string]bool{
```

Comment:
since the `bool` part of this map isn't actually used, I'd probably switch to using a set (`k8s.io/apimachinery/pkg/util/sets` -- see `sets.NewString(...)`).  Then see corresponding comments below